api/src/test/java/org/apache/iceberg/TestHelpers.java

```diff
@@ -402,6 +406,101 @@ public int hashCode() {
     }
   }
 
+  /** A VariantLike implementation for testing accepting JSON input */
+  public static class JsonVariant implements VariantLike {
```

Comment:
Why have a test implementation of a Variant rather than just adding real implementation? This doesn't seem valuable to me.

Reply:
The real implementation will be added to Iceberg-Parquet module which will use Parquet variant encoding/decoding (see https://github.com/aihuaxu/iceberg/pull/1/files#diff-7da5666792da1f7ea51905e40103482b66c749dda6a5c85c8a5530eda0832b54). This test implementation in Iceberg-api is only to test out the methods in VariantLike. Otherwise, we will introduce dependency to Parquet. Let me know if I understand correctly on how the code is orginized.

Reply:
Removed the implementation. Right now we are adding the type only.